tests/framework/pvc.go

```diff
@@ -35,9 +35,13 @@ func (f *Framework) CreatePVCFromDefinition(def *k8sv1.PersistentVolumeClaim) (*
 // on WaitForFirstConsumer storage class by executing f.ForceBindIfWaitForFirstConsumer(pvc)
 func (f *Framework) CreateBoundPVCFromDefinition(def *k8sv1.PersistentVolumeClaim) (*k8sv1.PersistentVolumeClaim, error) {
 	pvc, err := utils.CreatePVCFromDefinition(f.K8sClient, f.Namespace.Name, def)
-	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	if err != nil {
```

Comment:
Does it make sense to keep the Expects here, and simply not return an err at all? Less code everywhere else to check the err.

Reply:
Done